src/shadowbox/docker/Dockerfile

```diff
@@ -12,16 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1-alpine as outline-ss-server
-RUN apk add --no-cache git
-RUN go get github.com/Jigsaw-Code/outline-ss-server
-
 # See versions at https://hub.docker.com/_/node/
 FROM node:8.11.3-alpine
 
 # Versions can be found at https://github.com/shadowsocks/shadowsocks-libev/releases
 ARG SS_VERSION=3.2.0
 
+# Versions can be found at https://github.com/Jigsaw-Code/outline-ss-server/releases
+ARG SS_GO_VERSION=0.0.1
```

Comment:
We should start with 1.0.0. Now that we are using it, we can't break compatibility anymore.

Reply:
Sure, done.